astropy/nddata/bitmask.py

```diff
@@ -0,0 +1,411 @@
+            list of integer bit flags. When ``ignore_flags`` is either
+            `None` or a string list of flags, ``flip_bits`` **MUST** be set
+            to `None`.
+
+    good_mask_value : int, bool (Default = False)
+        This parameter is used to derive the values that will be assigned to
+        the elements in the output boolean mask array that correspond to the
+        "good" bit fields (that are 0 after zeroing bits specified by
+        ``ignore_flags``) in the input ``bitfield`` array. When
+        ``good_mask_value`` is non-zero or ``numpy.True_`` then values in the
+        output boolean mask array corresponding to "good" bit fields in
+        ``bitfield`` will be ``numpy.True_`` (if ``dtype`` is ``numpy.bool_``)
+        or 1 (if ``dtype`` is of numerical type) and values of corresponding
+        to "bad" flags will be ``numpy.False_`` (or 0). When
+        ``good_mask_value`` is zero or ``numpy.False_`` then the values
+        in the output boolean mask array corresponding to "good" bit fields
+        in ``bitfield`` will be ``numpy.False_`` (if ``dtype`` is
+        ``numpy.bool_``) or 0 (if ``dtype`` is of numerical type) and values
+        of corresponding to "bad" flags will be ``numpy.True_`` (or 1).
+
+    dtype : data-type (Default = ``numpy.bool_``)
+        The desired data-type for the output binary mask array.
+
+    Returns
+    -------
+    mask : numpy.ndarray
+        Returns an array of the same dimensionality as the input ``bitfield``
+        array whose elements can have two possible values,
+        e.g., ``numpy.True_`` or ``numpy.False_`` (or 1 or 0 for integer
+        ``dtype``) according to values of to the input ``bitfield`` elements,
+        ``ignore_flags`` parameter, and the ``good_mask_value`` parameter.
+
+    Examples
+    --------
+        >>> from astropy.nddata import bitmask
+        >>> import numpy as np
+        >>> dqbits = np.asarray([[0, 0, 1, 2, 0, 8, 12, 0],
+        ...                      [10, 4, 0, 0, 0, 16, 6, 0]])
+        >>> bitmask.bitfield_to_boolean_mask(dqbits, ignore_flags=0,
+        ...                                  dtype=int)
+        array([[0, 0, 1, 1, 0, 1, 1, 0],
+               [1, 1, 0, 0, 0, 1, 1, 0]])
+        >>> bitmask.bitfield_to_boolean_mask(dqbits, ignore_flags=0,
+        ...                                  dtype=bool)
+        array([[False, False,  True,  True, False,  True,  True, False],
+               [ True,  True, False, False, False,  True,  True, False]])
+        >>> bitmask.bitfield_to_boolean_mask(dqbits, ignore_flags=6,
+        ...                                  good_mask_value=0, dtype=int)
+        array([[0, 0, 1, 0, 0, 1, 1, 0],
+               [1, 0, 0, 0, 0, 1, 0, 0]])
+        >>> bitmask.bitfield_to_boolean_mask(dqbits, ignore_flags=~6,
+        ...                                  good_mask_value=0, dtype=int)
+        array([[0, 0, 0, 1, 0, 0, 1, 0],
+               [1, 1, 0, 0, 0, 0, 1, 0]])
```

Comment:
```suggestion
               [1, 1, 0, 0, 0, 0, 1, 0]]...)
```

I think this will fix the breaking tests